var/spack/repos/builtin/packages/attr/package.py

```diff
@@ -23,4 +31,7 @@ def configure_args(self):
     # Ref. http://www.linuxfromscratch.org/blfs/view/7.5/postlfs/attr.html
     def install(self, spec, prefix):
         make()
```

Comment:
Any idea why `make()` is present in the `install` phase? We already run it during the `build` phase.

Reply:
I've no idea about that, I think maybe it's a history issue :(

Reply:
If you remove it does it still install correctly?

Reply:
I tried remove it, and attr@2.4.47 and attr@2.4.48 can both build well.
And I uploaded this patch: https://github.com/spack/spack/pull/15535/commits/ad5d82a0cb72d25dc86bd53d4b3efee8186be3dd